docs/extend/plugin_api.md

```diff
@@ -16,7 +16,7 @@ Docker Engine.
 
 This page is intended for people who want to develop their own Docker plugin.
 If you just want to learn about or use Docker plugins, look
-[here](/extend/plugins).
+[here](plugins.md).
```

Comment:
@slevental Thank you for your contribution.  Can you test it using `make docs`? When I test this contribution it isn't fixing the problem.

This does:

```
 look [here](/extend/plugins).
```